### PR TITLE
Service setup

### PR DESCRIPTION
**Pull request by mcharrod**
I finally got the service reaching our first endpoint yayayaya

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -35,10 +35,24 @@ gem 'jbuilder', '~> 2.5'
 
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
+gem 'figaro'
+gem 'faraday'
```
Comment (Reuben-Davison) [inline Gemfile:39]: Lets remove one of the faradays

File: Gemfile [hunk 2]
```diff
@@ -35,10 +35,24 @@ gem 'jbuilder', '~> 2.5'
...
+  gem 'simplecov'
+  gem 'shoulda-matchers'
+  gem 'capybara'
+end
 
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
+  gem 'rspec-rails'
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'factory_bot'
```
Comment (Reuben-Davison) [inline Gemfile:52]: lets remove factory-bot because we needed factory_bot_rails

**Review by Reuben-Davison — CHANGES_REQUESTED** [review]
I think we can merge this after the gem file is cleaned up. The pre made comments in the RSpec Config could also be removed to make the code a bit cleaner

**Comment by alexGrandolph** [comment]
Katy could you add this line to the RecipesController

```class Api::V1::RecipesController < ApplicationController
  skip_before_action :verify_authenticity_token```
  
  For some reason its needed when trying to call the api from the front end

**Pull request merged**